### DISCUSSION TITLE
How to set packageClassName

### DISCUSSION THREAD

**Discussion by Saurabh29** [apache/netbeans#7688] [Q&A]
Dear All,

New to Apache NetBeans VSCode extension, when I try debugging a Java program many a times I get:

Error: Could not find or load main class ${packageClassName}
Caused by: java.lang.ClassNotFoundException: ${packageClassName}

Please let me know how to set packageClassName so that it can be passed to command.

Regards
Saurabh

**Comment by benbender** [comment]
I'm running into the same issue while trying to build and run a javacard-applet. Any help is greatly appreciated!
